app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -62,6 +62,7 @@ function DocumentCapture({ isAsyncForm = false, onStepChange = () => {} }: Docum
   const { inPersonURL, arcgisSearchEnabled } = useContext(InPersonContext);
   useDidUpdateEffect(onStepChange, [stepName]);
   useEffect(() => {
+    hijackSkipNav();
```

Comment:
does this need any guard to make sure it's not called twice? or should the function itself check to see that there's not already a listener set?

Reply:
To make sure I understand, is this what your concerned about? [Memory Issues](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#memory_issues)

If it is, I think it would solve the issue to create a function like: 
```
function setFocus(event) {
    event.preventDefault();
    mainContent?.focus();
    mainContent?.scrollIntoView();
  }
```


And then have the `addEventListener` work like `skipNavLink?.addEventListener('click', setFocus);`

Does that seem like a reasonable solution?

Reply:
@JackRyan1989 The return value to the `useEffect` callback should remove the event listener. Additionally, you may want `hijackSkipNav` to track the times it's been called/cleaned up to avoid redundant attachment (and premature detachment) of event listeners.

Reply:
yeah either solution works but I definitely think the unmount solution would be the cleanest